CMakeLists.txt

```diff
@@ -56,17 +62,23 @@ target_include_directories(spirv-cross-core PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 if (NOT "${MSVC}")
   set(spirv-compiler-options -std=c++11 -Wall -Wextra -Werror -Wshadow)
   set(spirv-compiler-defines __STDC_LIMIT_MACROS)
+
+  set(spirv-exe-options ${spirv-compiler-options})
```

Comment:
I think it makes sense to propagate -fno-exceptions even to executable to be able to test this more easily.